exercises/two-bucket/description.md

```diff
@@ -25,4 +25,4 @@ To conclude, the only valid moves are:
 - emptying one bucket and doing nothing to the other
 - filling one bucket and doing nothing to the other
 
-Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by [Lindsay](http://lindsaylevine.com).
+Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by Lindsay Levine.
```

Comment:
I'm getting a 404 from the fullstackacademy.com url.  An alternative address should be sought out or at least this one removed.

Reply:
The fullstackacademy url works for me

Reply:
I tried it again and it worked this time.  First few attempts were 404.

Reply:
odd